.github/actions/templates/publishModule/action.yml

```diff
@@ -77,6 +85,7 @@ runs:
 
         # Publish the modified child resources
         foreach ($ModuleToUpdate in $ModulesToUpdate) {
+          Write-Output "::group::- $($ModuleToUpdate.TemplateFilePath)"
```

Comment:
I'd almost argue to just have a grouping at the beginning and the end of the tasks. I mimics ADO + it leaves the Logs to the script

Reply:
It will not help us get to the specific outputs, but sure can do.